index.html

```diff
@@ -342,19 +342,19 @@ <h2>Open Event Organizer App</h2>
 	<div class="container">
 		<div class="row">
 			<div class="col-sm-4">
-				<h2 class="number">370+ students</h2>
+				<a href="https://gci17.fossasia.org/students"><h2 class="number">370+ students</h2></a>
 				<p>
 					Over 370 students participated in Fossasia Google code-in 2017, many students made their way in contributing on many open source projects.
 				</p>
 			</div>
 			<div class="col-sm-4">
-				<h2 class="number">300+ Tasks</h2>
+				<a href="https://codein.withgoogle.com/tasks/?sp-organization=6237166344601600"><h2 class="number">300+ Tasks</h2></a>
```

Comment:
Replace link with this - https://codein.withgoogle.com/organizations/fossasia/

Reply:
ohkk